cmake/CMakeLists.txt

```diff
@@ -1,13 +1,35 @@
+# Minimum CMake required
 cmake_minimum_required(VERSION 2.8)
 
+# Project
 project(protobuf C CXX)
 
+# Options
 option(BUILD_TESTING "Build tests" ON)
 option(BUILD_SHARED_LIBS "Build Shared Libraries" OFF)
 if (MSVC)
   option(ZLIB "Build with zlib support" OFF)
 endif (MSVC)
 
+# Path to common header
+set(protobuf_COMMON_HEADER "../src/google/protobuf/stubs/common.h")
```

Comment:
Extract the version number from configure.ac instead?
https://github.com/google/protobuf/blob/master/configure.ac#L15

common.h only has version number like 3000000, but we probably want something like 3.0.0-alpha-4.